src/config.rs

```diff
@@ -324,6 +325,18 @@ impl LLMConfig {
     }
 }
 
+impl Debug for LLMConfig {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("OrganizationCredentials")
+            .field("identifier", &self.identifier)
+            .field("api_key", &"****")
```

Comment:
Should we use `.map(|_| "****".to_string()` on the`api_key`, given it's an `Option`?

Reply:
Yes we should, I forgot it was optional